tests/bin/test_boot_basic.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
```

Comment:
I don't think any of the .py files in `bin/` need to be executable, and therefore don't need a shebang line. Unless that is a checkbox requirement? Pytest will just scan files for functions beginning with `test` and run them.

Reply:
Unresolving because now that the files are non-executable, the directory name `bin/` should probably be changed. Maybe `pytest/` instead?